include/bluetooth/services/ancs_client.rst

```diff
@@ -37,32 +37,32 @@ Once a connection is established with a central device, the module needs a servi
 If this succeeds, the handles for the ANCS server must be assigned to an ANCS client instance using the :c:func:`bt_ancs_handles_assign` function.
 
 The application can now subscribe to iOS notifications with :c:func:`bt_ancs_subscribe_notification_source`.
-The notifications arrive in the :c:enumerator:`BT_ANCS_EVT_NOTIF` event.
+The notifications arrive through callback function of the notification source subscription.
 Use :c:func:`bt_ancs_request_attrs` to request attributes for the notifications.
-The attributes arrive in the :c:enumerator:`BT_ANCS_EVT_NOTIF_ATTRIBUTE` event.
+The attributes arrive through callback function of the data source subscription with a Command ID :c:enumerator:`BT_ANCS_COMMAND_ID_GET_NOTIF_ATTRIBUTES`.
 Use :c:func:`bt_ancs_request_app_attr` to request attributes of the app that issued the notifications.
```

Comment:
```suggestion
Use :c:func:`bt_ancs_request_app_attr` to request attributes of the application that issued the notifications.
```

Reply:
The app refers to an iPhone app. Let's stick to the Apple terminology.

Reply:
Fair enough.